pkg/sql/schemachanger/scexec/scmutationexec/constraint.go

```diff
@@ -122,14 +122,7 @@ func (i *immediateVisitor) MakeValidatedCheckConstraintPublic(
 		if c := mutation.GetConstraint(); c != nil &&
 			c.ConstraintType == descpb.ConstraintToUpdate_CHECK &&
 			c.Check.ConstraintID == op.ConstraintID {
-			// Remove the mutation from the mutation slice. The `MakeMutationComplete`
-			// call will mark the check in the public "Checks" slice as VALIDATED.
-			err = tbl.MakeMutationComplete(mutation)
```

Comment:
Here and elsewhere in this file I just inlined this method and found to result to be simpler and less error-prone. In any case this was failing when reverting a DROP CONSTRAINT schema change, MakeMutationComplete would choke on the fact that the mutation was ADD and the validating state was Dropping.